buildlib/jucx-publish.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+  target: publish
+  temp_cfg: bindings/java/src/main/native/build-java/tmp-settings.xml
+
+steps:
+  - bash: ./autogen.sh
+    displayName: Setup autotools
+
+  - bash: |
+      set -eE
+      ./contrib/configure-release --with-java
+      make -s -j`nproc`
```

Comment:
where are we building the tarball and src rpm?
@amaslenn AFAIK --with-java should not have effect on the dist tarball created by "make dist", we publish pre-autogen.sh-ed tarball, not pre-configured tarball..

Reply:
Ahh, it is missing now. OK, if `--with-java` doesn't affect the tarball, there are no changes here.
@petro-rudenko please add `./contrib/buildrpm.sh -s -t -b` as a separate step after `jucx-publish` in release pipeline.